### DISCUSSION TITLE
Installation of 8.0 was great...but my account was locked at the gitgo

### DISCUSSION THREAD

**Discussion by DarcSceptor0624** [elementary/installer#856] [Q&A]
System:
Minisforum 790S7  
  AMD Ryzen 9 7940HX 
  32G DDR5 RAM    1TB Nvme 4.0 
  RTX 4060 GPU

I installed Elementary and I must say it went very well. I like that it goes right into installation.
After installation it collected some redundant information and it then got my user info.
I entered my name which became my login as well. I entered t1m3l1n3 as my password because I can quickly access admin functions but your software said it failed a dictionary test. All password fields were red. But I thought I'd see if it would be accepted regardless. Well it took the password, it came up to login, and.....MY ACCOUNT WAS LOCKED. The only account that can enter the OS and it was locked. 

So then I swore and tried rebooting but still it went to the lock screen. So I had to reboot and reinstall. But that led to the install drive being locked up(??) and could not reinstall. I finally had to get my Linux Mint install which let me reformat and reinstall Mint.

1) How the hell can t1m3l1n3 fail a dictionary check?
2) If it ends up locking my account WHY let me move beyond the password creation screen?

Expected Results
While I cherish secure systems this is not a server. It is my person system. If I want a weak password then I should be allowed to enter a weak password. After all I have a boot knife sitting under my desk if an intruder wanted access to my computer.

If your rules are just 'do not allow' then for God Sake do not let me move off the screen until I have a password that will let me actually log into the system. If you have a backdoor login you should tell me, the installer, so I can use it. However clicking on my name did not expose any alternative ID I could enter or select. I was SCREWED.

**Comment by danirabbit** [comment]
Hey thanks for your feedback. I'm going to close this discussion because it looks like you already filed an issue report which is probably the more appropriate method here: https://github.com/elementary/initial-setup/issues/202
